.github/workflows/community-tutorial-checks.js.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 The Google Earth Engine Community Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This workflow will do a clean install of node and run tests.
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Community Tutorial checks
```

Comment:
IIUC this is intended to run checks on the JS Code Editor community tutorials?
If so, I wonder if the name should include some indicator of "JS" to make it more clear and delineate from a (future?) IPYNB check.

Reply:
These tests actually run against both. I inherited the *.js.* from the examples, but I suppose it's not relevant here. I'll rename instead.

Reply:
Continuing comment from https://github.com/google/earthengine-community/pull/364#discussion_r654590176, it's worth noting that this file could (if desired) contain multiple jobs, i.e. one that checks the JS tutorials and another for IPYNB. So the workflow could be given a generic name ("CI") and have jobs with more specific names ("JS Tutorials", "PY Tutorials").

That said... I'm fine with either approach, please feel free to ignore my earlier comment. :)

Reply:
SGTM